scrapy/extensions/httpcache.py

```diff
@@ -366,7 +365,7 @@ def parse_cachecontrol(header):
 
 def rfc1123_to_epoch(date_str):
     try:
-        date_str = to_unicode(date_str, encoding='ascii')
+        date_str = str(date_str, encoding='ascii')
```

Comment:
this would fail if date_str is already str, so technically this is a backwards incompatible change